web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -123,6 +132,67 @@ be edited, its `Revision: X` label will be increased by one, and a note will be
+### Clear examples for potential PDEPs:
+- Adding a new parameter to many existing methods, or deprecating one in many places. For example:
+  - The `numeric_only` deprecation affected many methods and could have been a PDEP.
+- Adding a new data type has impact on a variety of places that need to handle the data type.
+  Such wide-ranging impact would require a PDEP. For example:
+  - `Categorical` ([GH-7217][7217], [GH-8074][8074]), `StringDtype` ([GH-8640][8640]), `ArrowDtype`
+- A significant (breaking) change in existing behavior. For example:
+  - Copy/view changes ([GH-36195][36195])
+- Support of new python features with a wide impact on the project. For example:
+  - Supporting typing within pandas vs. creation of `pandas-stubs` ([GH-43197][43197],
+    [GH-45253][45253])
+- New required dependency
+
+### Borderline examples:
+- Small changes to core functionality, such as `DataFrame` and `Series`, should always be considered
+  as a PDEP candidate as it will likely have a big impact on users. But the same types of
+  changes in other methods would not be good PDEP candidates. That said, any discussion, no
+  matter how small the change, which becomes controversial is a PDEP candidate. Consider if more
+  attention and/or a formal decision-making process would help. Following are some examples we
+  hope can help clarify our meaning here:
+- API breaking changes, or discussion thereof, could be a PDEP. For example:
+  - Value counts rename ([GH-49497][49497]). The scope doesn’t justify a PDEP at first, but later a
+    discussion about whether it should be executed as breaking change or with deprecation
+    emerges, which could benefit from a PDEP process.
+- Adding new parameters or methods to an existing method typically won't require a PDEP for
+  non-core features. For example:
+  - Both `dropna(percentage)` ([GH-35299][35299]), and `Timestamp.normalize()` ([GH-8794][8794])
+    wouldn't have required a PDEP.
+  - On the other hand, `DataFrame.assign()` might. While it's a single method without backwards
+    compatibility concerns, it’s also a core feature and the discussion should be highly visible.
+- Deprecating or removing a single method would not require a PDEP in most cases. For example:
+  - `DataFrame.xs` ([GH-6249][6249]) is an example of deprecations on core features that would be
+    good a candidate for a PDEP.
+- Changing the default value of parameters in a core pandas method is another edge case. For
+  example:
+  - Such changes in `dropna`,  `DataFrame.groupby`, or in `Series.groupby` could be PDEPs.
+- New top level modules and/or exposing internal classes. For example:
+  - Add `pandas.api.typing` ([GH-48577][48577]) is relatively small and wouldn’t necessarily
+    require a PDEP.
+
+### A counter-example:
+- Significant changes to contributors' processes don't require a PDEP as they aren't going to
+  have an impact on users. For example:
+  - Changing the build system to meson ([GH-49115][49115])
```

Comment:
We did have a PDEP for that, and I think it was quite useful. What do you think @WillAyd

Reply:
Yea I think this was worth a PDEP - it definitely falls under the category of a significat `internal change`

Reply:
thanks! 🙇

Reply:
Would you say such process changes would be boreder-line cases or clear PDEP examples? I'm wondering where to move this example too

Reply:
I think this one is clear. If it's a significant change to the developer workflow I think it could benefit from a PDEP whether user facing or not. But no hard opinion it others disagree